app/views/support_interface/performance/feature_metrics_dashboard.html.erb

```diff
@@ -242,5 +242,31 @@
         </div>
       </div>
     </div>
+    <div class="govuk-grid-column-one-half">
+      <div id="headline-stat" class="govuk-!-margin-bottom-4">
+        <%= render SupportInterface::TileComponent.new(
+          count: @dashboard.read_metric(:apply_again_change_rate),
+          label: 'candidates that make changes to application when applying again',
```

Comment:
super nitty, but probably should be 'made changes to their application'. Sorry to be that guy haha